src/tasks/onetime/TakeDeliveryTask.py

```
@@ -60,7 +60,7 @@ def process_ocr_results(self, full_texts, filter_min, reward_pattern):
                             rewards.append((t, val))  # 保存 OCR对象 和 提取出的数值
                         elif val > 100:
                             self.log_debug(f"金额异常过大({val}万)，可能是OCR误识别，已过滤")
-                    except:
+                    except Exception:
                         pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)